backend/tests/test_auth_real.py

```
@@ -1,60 +1,651 @@
+    token = _signed_session_token(_valid_session_payload(), repeated_secret)
+    settings.AUTH_SESSION_HMAC_SECRET = SecretStr(repeated_secret)
+
+    with pytest.raises(HTTPException) as exc:
+        await get_auth_context(authorization=f"Bearer {token}")
+
+    assert exc.value.status_code == 401
+
+
+@pytest.mark.asyncio
+async def test_signed_bearer_session_rejects_public_fixture_secret():
+    public_fixture_secret = "naruon-session-hmac-token-32-byte-minimum"
```

Ruff (0.15.13)

[error] Possible hardcoded password assigned to: "public_fixture_secret"

(S105)